### DISCUSSION TITLE
Slack Channel invite link no longer valid

### DISCUSSION THREAD

**Discussion by labkey-jony** [cloudnative-pg/cloudnative-pg#2818] [Q&A] [answered]
Hi Everyone,

I'm trying to access the Slack channel from the invite link on the README file:

https://join.slack.com/t/cloudnativepg/shared_invite/zt-17culux7k-P_UsVOOR9teUYi4dGhDSBQ

But it appears to no longer be active.

![Screenshot 2023-09-19 at 11 35 16 AM](https://github.com/cloudnative-pg/cloudnative-pg/assets/10408922/18afc7fe-d2fc-4d98-8cae-d1ed281b6529)

Can someone setup a new invite URL on the README?

**Comment by armru** [comment, marked as answer]
Hello, 

Does https://github.com/cloudnative-pg/cloudnative-pg/commit/66883a12a11c102b2dd7a0b12a9c0ef6764bce1d fix the issue?

**Reply by labkey-jony** [reply]
Confirmed! I'm in!

**Comment by thekoma** [comment]
Again!

**Comment by sxd** [comment]
@thekoma can you share the message? the link was updated a couple of weeks ago

**Reply by thekoma** [reply]
I'm on iphone so the message is localized in app, but it says that is expired.

**Reply by sxd** [reply]
I just tested with a friend that it's working, sorry but are you actually clicking the new link ?

**Comment by thekoma** [comment]
Tested it in a Incognito window using email instead of bneing logged into Slack and worked. 
Cleaned cache and logged back in my account and then tested again the invite and didn't work again. 
I suspect Slack is having a bad day. :)
"Fixed"
